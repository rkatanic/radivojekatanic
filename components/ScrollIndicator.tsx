import Menu from "./Menu";

const ScrollIndicator = (): JSX.Element => (
  <div className="absolute -left-48 w-40 dark:text-neutral-200">
    <Menu />
  </div>
);

export default ScrollIndicator;
