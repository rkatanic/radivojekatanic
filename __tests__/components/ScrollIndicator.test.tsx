import ScrollIndicator from "@/components/ScrollIndicator";
import { render } from "@testing-library/react";

describe("ScrollIndicator", (): void => {
  it("should render", (): void => {
    const { baseElement } = render(<ScrollIndicator />);

    expect(baseElement).toMatchSnapshot();
  });
});
