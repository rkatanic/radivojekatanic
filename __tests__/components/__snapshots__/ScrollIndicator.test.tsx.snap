// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollIndicator should render 1`] = `
<body>
  <div>
    <div
      class="hidden lg:flex absolute top-0 -left-16 h-full flex-col justify-between font-medium dark:font-normal"
    >
      <a
        class="group flex items-end gap-2 w-0 h-0 mt-10 -rotate-90 dark:text-neutral-600 text-sm text-neutral-300 dark:hover:text-neutral-200 hover:text-neutral-900"
        href="#bottom"
        id="top"
      >
        <div
          class="group-hover:bg-neutral-900 dark:group-hover:bg-neutral-200 bg-neutral-300 dark:bg-neutral-600 w-6 h-px absolute -top-[0.55rem] -left-8"
        />
        <div>
          scroll
        </div>
      </a>
      <a
        class="group flex items-end gap-2 w-0 h-0 mt-10 -rotate-90 dark:text-neutral-600 text-sm text-neutral-300 dark:hover:text-neutral-200 hover:text-neutral-900"
        href="#"
        id="bottom"
      >
        <div
          class="group-hover:bg-neutral-900 dark:group-hover:bg-neutral-200 bg-neutral-300 dark:bg-neutral-600 w-6 h-px absolute -top-[0.65rem] left-12"
        />
        <div>
          scroll
        </div>
      </a>
    </div>
  </div>
</body>
`;
